ksql-engine/src/main/java/io/confluent/ksql/exception/KafkaDeleteTopicsException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.exception;
+
+import io.confluent.ksql.util.KsqlException;
+import java.util.List;
+import javafx.util.Pair;
+
+public class KafkaDeleteTopicsException extends KsqlException {
```

Comment:
extend `KafkaTopicClientExceptionHere` instead of `KsqlException`. `KsqlException` _should_ really only be used for errors indicating a problem with the user's statement (though we need to clean this up in places).